pkg/proxy/proxyports/proxyports_test.go

```diff
@@ -278,8 +279,8 @@ func TestRestoredPort(t *testing.T) {
 	require.Zero(t, pp.nRedirects)
 
 	// wait for port reuse wait to pass
-	time.Sleep(time.Millisecond)
```

Comment:
I'd like to know why this slipped through review; `time.Sleep()` is notorious for causing this kind of problem and I personally think we should have a near zero tolerance for it during review.

Reply:
Ref https://github.com/cilium/cilium/pull/36953 , something to consider for future @jrajahalme @aanm @sayboras .

Reply:
hope that the coming syntest in go 1.24 can help in this case, we can enable linter to disallow time.Sleep, the author can bypass if required.